src/lobby/lobbyapp.cpp

```diff
@@ -735,66 +735,38 @@ bool CLobbyApp::GetRankForCallsign(const char* szPlayerName, int *rank, double *
 {
```

Comment:
is this function still being used? isn't ACSS gone?

Reply:
ACSS is gone, and I just checked the server, that registry for the lobby points to the old ACSS server and is for sure not going anywhere. So, you can remove this. Thanks!